hector_mapping/src/HectorMappingRos.cpp

```diff
@@ -392,15 +415,7 @@ bool HectorMappingRos::resetMapCallback(std_srvs::Trigger::Request  &req,
 bool HectorMappingRos::pauseMapCallback(std_srvs::SetBool::Request  &req,
                                         std_srvs::SetBool::Response &res)
 {
-  if (req.data && !pause_scan_processing_)
-  {
-    ROS_INFO("HectorSM Mapping paused");
-  }
-  else if (!req.data && pause_scan_processing_)
-  {
-    ROS_INFO("HectorSM Mapping no longer paused");
-  }
-  pause_scan_processing_ = req.data;
+  this->toggleMappingPause(req.data);
```

Comment:
because this chunk of code would be identical in `hectorManagementCallback` and `pauseMapCallback`, I created the function `toggleMappingPause` that has the very same chunk of code